broker/src/main/java/io/camunda/zeebe/broker/system/SystemContext.java

```diff
@@ -192,6 +192,7 @@ private void validateBackupCfg(final BackupStoreCfg backup) {
                 .withApiCallTimeout(s3Config.getApiCallTimeout())
                 .forcePathStyleAccess(s3Config.isForcePathStyleAccess())
                 .withCompressionAlgorithm(s3Config.getCompression())
+                .withBasePath(s3Config.getBasePath())
                 .build();
         S3BackupStore.validateConfig(storeConfig);
```

Comment:
:X: In `S3BackupStore.validateConfig` please validate that `basePath` is valid. I missed this in the last PR - you can also move the validations in the S3BackupConfig constructor here so all validations are in the same place.